bin/kafka_broker_generator.py

```diff
@@ -149,15 +196,17 @@
 
 """
 
+with open("payloads/payload.68KB.txt") as f:
+    payload = f.read().replace("\n", "")
```

Comment:
I really like this,

Reply:
yes, eventually the file path will be an argument of the generator